### ISSUE TITLE
Alert and notification list page

### ISSUE THREAD

**Issue by gnehapk** [Tendrl/dashboard#304]

**Comment by gnehapk** [comment]
@anmolbabu Please provide the property matched against cluster_name and fqdn in alert list API response.

@anivargi Please provide the notification list API response.

**Comment by gnehapk** [comment]
@julienlim @mcarrano I have few questions regarding the event management UX designs- 

1) Is the list filtered by clicking the severity icons(warning,error, info)? If yes, then how to display that this filter is applied and how to clear it too? UX - https://redhat.invisionapp.com/share/8N93NO7Q4#/screens/200318244/
2) How the UI will decide that the filter(for eg cluster: cluster 1) needs to display in active filter list? There should be some button after filter by input, which tells to add in the active filter list and hence the filtering of list will be done. UX - https://redhat.invisionapp.com/share/8N93NO7Q4#/screens/200318241/
3) How the active filters will be shown on the UI, if the length exceeds the screen width? Or there is any upper limit of filters that can be applied here? UX - https://redhat.invisionapp.com/share/8N93NO7Q4#/screens/200318241/

**Comment by mcarrano** [comment]
I have responded to each of these issues within InVision, but here is a quick summary.

1. Applying a filter for severity will follow the same pattern as any other filter.  I've added more detail in the annotation here: https://redhat.invisionapp.com/share/8N93NO7Q4#/200318244_Admin-Events
2. The filter is activated by pressing the Enter key per PatternFly.  See interaction design details here http://www.patternfly.org/pattern-library/forms-and-controls/filter/#/design
3. There is no upper limit on the number of filters that can be applied.  The content of this row should simply wrap.  I've provided a link to a sample PatternFly implementation in the InVision comment.

I have also made some updates to the annotations on this page to provide more clarity. Let me know if you have further questions @gnehapk
